client/src/components/Contact.style.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
```

Comment:
it's often best not to set a height, but to let the content fill the page. 
I would get rid of this ```height: 100vh``` and then use margin/padding to better space your elements!